x-pack/plugins/case/common/api/user.ts

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import * as rt from 'io-ts';
+
+export const UserRT = rt.type({
+  full_name: rt.union([rt.undefined, rt.string, rt.null]),
```

Comment:
we need to try for email too, which is potentially undefined. can be done in a follow up